Gemfile.lock

```diff
@@ -193,7 +194,7 @@ GEM
       thread_safe (~> 0.3, >= 0.3.1)
     barby (0.6.8)
     base32-crockford (0.1.0)
-    base64 (0.1.1)
+    base64 (0.2.0)
```

Comment:
so last time we upgraded rails based on snyk, we saw this error when booting the app in deployed environments:
```
App 9517 output: Error: The application encountered the following error: You have already activated base64 0.1.1, but your Gemfile requires base64 0.2.0. Since base64 is a default gem, you can either remove your dependency on it or try updating to a newer version of bundler that supports base64 as a default gem. (Gem::LoadError)
```

so we ended up reverting it
https://github.com/18F/identity-idp/pull/9577 & https://github.com/18F/identity-idp/pull/9606

Have we tried this branch in a deployed env? if not, could we pin `base64` in the meantime?

Reply:
https://github.com/18F/identity-devops/pull/6832 should fix it